hawc/main/settings/base.py

```diff
@@ -93,6 +93,8 @@
     "django.contrib.sites",
     "django.contrib.messages",
     "django.contrib.staticfiles",
+    "dal",
```

Comment:
move these into the "external apps" section; honestly while we're at it we should move django_filters down too

Reply:
It's suggested to include these before `django.contrib.admin`, which is why its up here and not in the "external apps" section:
https://django-autocomplete-light.readthedocs.io/en/master/install.html#configuration